specification/sql/resource-manager/Microsoft.Sql/preview/2024-05-01-preview/DatabaseAdvisors.json

```diff
@@ -406,9 +406,7 @@
         "details": {
           "description": "Gets additional details specific to this recommended action.",
           "type": "object",
-          "additionalProperties": {
-            "type": "object"
-          },
+          "additionalProperties": true,
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

This seems to be equivalent.

Did you mean to set the type as string ?

Reply:
Setting the type as string was my first idea as well, but the additional properties are not always just string, sometimes they are other types like integer and so on. And guessing on the ARM swagger validation check that was failing, I couldn't use type object, as other types were present. It's not equivalent to just additionalProperties: true. So I just changed it to the most simplest solution, where both string and integer and any other type can be taken in.